services/cloudsecuritygroup/cloudsecuritygroup.go

```diff
@@ -0,0 +1,6 @@
+package cloudsecuritygroup
+
+const (
+	RESTfulPathAWS   = "CloudSecurityGroup"
+	RESTfulPathAzure = "AzureSecurityGroupPolicy"
```

Comment:
The name of the constant I think that it should be RESTfulPathAzureSecurityGroup because in cloudaccounts constants we use the same name to represent "AzureCloudAccount", /i think this is little bit confusing